.gitignore

```diff
@@ -16,7 +16,6 @@ artifacts/
 # Visual Studio workspace state
 .vs/
 
-# Rush files
-/common/temp/**
-package-deps.json
-**/.rush/temp
```

Comment:
What are these files and why do we not need them in the .gitignore anymore? 🤔

Reply:
A few years ago, we were using [rush](https://rushjs.io/) instead of npm. For our purposes, rush was overkill, so we switched to npm. We just never removed these ignore files.